easybuild/easyconfigs/z/Z3/Z3-4.15.1-GCCcore-14.3.0.eb

```
@@ -0,0 +1,48 @@
+easyblock = 'PythonBundle'
+
+name = 'Z3'
+version = '4.15.1'
+
+homepage = 'https://github.com/Z3Prover/z3'
+description = """Z3 is a theorem prover from Microsoft Research with support for bitvectors,
+booleans, arrays, floating point numbers, strings, and other data types. This
+module includes z3-solver, the Python interface of Z3.
+"""
+
+toolchain = {'name': 'GCCcore', 'version': '14.3.0'}
+
+builddependencies = [
+    ('binutils', '2.44'),
+    ('CMake', '4.0.3'),
+]
+
+dependencies = [
+    ('Python', '3.13.5'),
+    ('GMP', '6.3.0'),
+]
+
+
+_fix_parallelism = """sed -i 's/str(multiprocessing.cpu_count())/"%(parallel)s"/' setup.py && """
+_enable_gmp = """sed -i "s/Z3_USE_LIB_GMP.*/Z3_USE_LIB_GMP' : True,/" setup.py && """
+
+exts_list = [
+    ('z3_solver', '4.15.1.0', {
+        'modulename': 'z3',
+        'preinstallopts': """sed -i 's/str(multiprocessing.cpu_count())/"%(parallel)s"/' setup.py && sed -i "s/Z3_USE_LIB_GMP.*/Z3_USE_LIB_GMP' : True,/" setup.py && """,
```

```suggestion
        'preinstallopts': _fix_parallelism + _enable_gmp,,
```

Looks like `--inject-checksums` expanded this variable.